packages/@webex/plugin-meetings/src/media/index.ts

```diff
@@ -163,6 +163,19 @@ Media.createMediaConnection = (
   // we might not have any TURN server if TURN discovery failed or wasn't done or
   // we might get an empty TURN url if we land on a video mesh node
   if (turnServerInfo?.url) {
+    if (!isBrowser('firefox')) {
+      let bareTurnServer = turnServerInfo.url;
+      bareTurnServer = bareTurnServer.replace('turns', 'turn');
```

Comment:
maybe better to have a more specific search term like "turns:" just in case if we end up with "turns" string being part of the hostname

Reply:
Done